scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -146,6 +155,36 @@ def get_cluster(self):
     check_type(cluster, dict, "cluster")
     return cluster
 
+def get_node(self):
+    """Get node"""
+    try:
+        node = Conf.get(self._index, 'node')
+    except:
+        raise MotrError(errno.EINVAL, "node does not exist in ConfStore")
+
+    check_type(cluster, dict, "node")
+    return node
+
+def get_cortx(self):
+    """Get cortx"""
+    try:
+        cortx = Conf.get(self._index, 'cortx')
+    except:
+        raise MotrError(errno.EINVAL, "cortx does not exist in ConfStore")
+
+    check_type(cluster, dict, "cortx")
+    return cortx
+
+def get_data(self, dname, tname):
```

Comment:
replace dname to key.
replace tname to key_type

Reply:
Done ed4d648